acceptance/extension-logging-fluentd/src/__tests__/accpetance/fluent.acceptance.ts

```diff
@@ -11,25 +11,28 @@ import {readLog} from '../fixtures/fluentd.docker';
 
```

Comment:
i think there's a typo in this folder name.

Reply:
Fixed now.